### ISSUE TITLE
[syncfusion_flutter_charts] Custom trackball tooltip not showing, when not fitting into chart

### ISSUE THREAD

**Issue by WieFel** [syncfusion/flutter-widgets#495]
Hi,
I am trying to implement a custom trackball tooltip using `TrackballBehavior.builder` and `tooltipDisplayMode: TrackballDisplayMode.groupAllPoints`.
Everything works fine, but I cannot make the tooltip too wide, as it simply disappears when it doesn't fit inside the chart width.
For example, in a case where I have 3 series (with 3 y-axes showing), the actual chart draw area is very small. Thus, the tooltip is not showing at all in some cases. The problem occurs regarless of which `tooltipAlignment` I set.

https://user-images.githubusercontent.com/8345062/146538821-b8261101-7270-457a-b73d-6e67416af308.mov

Is there any way to let the tooltip at least overlap also the y-axes, and this way provide more space for it?
Or is there any other solution, to display the tooltip as a sort of overlay, which shows above the whole screen, regardless of being within the chart area or not?

**Comment by Yuvaraj-Gajaraj** [comment]
Hi @WieFel,

Greetings from Syncfusion. We have analyzed your query, and we have considered this behavior as an improvement, and it will be implemented in any of our upcoming releases yet to come. We appreciate your patience until then, you can track the feature status in the below link.

Feedback: https://www.syncfusion.com/feedback/31338 

Regards,
Yuvaraj.

**Comment by WieFel** [comment]
@Yuvaraj-Gajaraj any news on this?

We would want to ship our charts to production already, but the problem of the tooltip being invisible sometimes makes it impossible for us to ship that to clients.

As a temporary solution, I would suggest being able to set some fixed position (alignment) for the tooltip: e.g. topLeft of the whole chart area (overlapping y-axis), topCenter, topRight, center, bottomRight, ...
Basically the tooltip should just float over the chart on some fixed position, while moving the trackball.

Would that be possible?

**Comment by Yuvaraj-Gajaraj** [comment]
Hi WieFel,

We would like to let you know that the improvement for positioning the trackball tooltip will be available in our upcoming vol 1 2022 release which is expected to be rolled out at the end of March 2022.  And you can track the status of the improvement in the below feedback

https://www.syncfusion.com/feedback/31338 

Regards,
Yuvaraj.
